x-pack/test/api_integration/apis/ml/results/get_partition_fields_values.ts

```diff
@@ -0,0 +1,247 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { Datafeed, Job } from '@kbn/ml-plugin/common/types/anomaly_detection_jobs';
+import { USER } from '../../../../functional/services/ml/security_common';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { COMMON_REQUEST_HEADERS } from '../../../../functional/services/ml/common_api';
+
+export default ({ getService }: FtrProviderContext) => {
+  const esArchiver = getService('esArchiver');
+  const supertest = getService('supertestWithoutAuth');
+  const ml = getService('ml');
+
+  function getJobConfig(jobId: string, enableModelPlot = true) {
+    return {
+      job_id: jobId,
+      description:
+        'mean/min/max(responsetime) partition=airline on farequote dataset with 1h bucket span',
+      groups: ['farequote', 'automated', 'multi-metric'],
+      analysis_config: {
+        bucket_span: '1h',
+        influencers: ['airline'],
+        detectors: [
+          { function: 'mean', field_name: 'responsetime', partition_field_name: 'airline' },
+          { function: 'min', field_name: 'responsetime', partition_field_name: 'airline' },
+          { function: 'max', field_name: 'responsetime', partition_field_name: 'airline' },
+        ],
+      },
+      data_description: { time_field: '@timestamp' },
+      analysis_limits: { model_memory_limit: '20mb' },
+      model_plot_config: { enabled: enableModelPlot },
+    } as Job;
+  }
+
+  function getDatafeedConfig(jobId: string) {
+    return {
+      datafeed_id: `datafeed-${jobId}`,
+      indices: ['ft_farequote'],
+      job_id: jobId,
+      query: { bool: { must: [{ match_all: {} }] } },
+    } as Datafeed;
+  }
+
+  async function createMockJobs() {
+    await ml.api.createAndRunAnomalyDetectionLookbackJob(
+      getJobConfig('fq_multi_1_ae'),
+      getDatafeedConfig('fq_multi_1_ae')
+    );
+
+    await ml.api.createAndRunAnomalyDetectionLookbackJob(
+      getJobConfig('fq_multi_2_ae', false),
+      getDatafeedConfig('fq_multi_2_ae')
+    );
+  }
+
+  describe('GetAnomaliesTableData', function () {
+    before(async () => {
+      await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/ml/farequote');
+      await ml.testResources.setKibanaTimeZoneToUTC();
+      await createMockJobs();
+    });
+
+    after(async () => {
+      await ml.api.cleanMlIndices();
+    });
+
+    it('should fetch anomalous only field values within the time range with an empty search term sorting by anomaly score', async () => {
```

Comment:
Can we group these first tests in a `when model plot is enabled` suite so we get naming consistency with the second part (`when model plot is disabled`)?

Reply:
added in https://github.com/elastic/kibana/pull/133519/commits/ea17d6e25fe523aee11a661faecf669095b4ff62